frontend/src/SDKInitialize.tsx

```diff
@@ -36,7 +37,17 @@ const SDKInitialize: React.FC = ({ children }) => {
     }, 1000);
   }, []);
 
-  return <AppInitSDK configurations={config}>{ready ? children : null}</AppInitSDK>;
+  return (
+    <AppInitSDK configurations={config}>
+      {ready ? (
+        children
+      ) : (
+        <Bullseye>
+          <Spinner />
+        </Bullseye>
```

Comment:
Juntao noted this -- we should always show spinners now... this was really old code so we had not made that change yet and I forgot to review it again once I started rebasing.